src/Components/UI/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState, useEffect } from "react";
+import CarouselList from "./CarouselList";
+import "./Carousel.css";
+import { LeftArrow, RightArrow } from "../../../Assets/Icons/icons";
+
+const Carousel = ({ brandData }) => {
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  const nextSlide = () => {
+    currentIndex === brandData.length - 1
+      ? setCurrentIndex(0)
+      : setCurrentIndex(currentIndex + 1);
+  };
+
+  const prevSlide = () => {
+    currentIndex === 0
+      ? setCurrentIndex(brandData.length - 1)
+      : setCurrentIndex(currentIndex - 1);
+  };
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      nextSlide();
+    }, 2000);
+    return () => clearInterval(interval);
+  }, [currentIndex]);
```

Comment:
I found this helpful. Love your UI and how neat your code is! Great work, Muskan!!